### ISSUE TITLE
Enable CORS on data links

### ISSUE THREAD

**Issue by zzzev** [voteview/WebVoteView#356]
First, let me say I have a ton of appreciation for this project, thank you for existing and working on these issues.

It would be great if the data available [here](https://voteview.com/data) had CORS enabled so they could be used directly in the browser (for example in an [Observable notebook](//observablehq.com)).

More context on why this is a good idea [here](https://enable-cors.org/).

**Comment by aaronrudkin** [comment]
Good idea. I'm not sure for an ETA, but it's a fair point and I don't see any resource constraints that should prohibit us from doing this.
